### ISSUE TITLE
Remove obsolete HP ID xref (3)

### ISSUE THREAD

**Issue by asmvernon** [monarch-initiative/mondo#1180]
**Mondo term (ID and Label):**
MONDO:0018177

**Xref that should be fixed (ID and label):**
Remove HP:0100843; it is an obsolete ID

**Other comments:**

**Comment by nicolevasilevsky** [comment]
This term  is not obsolete in HPO:
https://hpo.jax.org/app/browse/term/HP:0012174
